### ISSUE TITLE
Presenter notes

### ISSUE THREAD

**Issue by gaborcsardi** [MangoTheCat/rmdshower#20]
Not quite sure how exactly but there are some possibilities, e.g.
https://www.christianheilmann.com/2012/08/15/browsers-have-a-presenter-mode-console-info/

**Comment by sje30** [comment]
Yes, would be great to have a separate presenter notes display on your laptop whilst the data projector shows your slides.  Other rmarkdown based slides have this, e.g. ioslides.  Most useful for me is a reminder of what slide is coming next.

**Comment by gaborcsardi** [comment]
@sje30 You can open two browser windows, one on each screen, and make the one on the projector full screen, and the one on the laptop in "slide view". 

It is not ideal, because you need to scroll the "slide view" on the laptop screen occasionally, but maybe it does the trick until rmdshower gets proper presenter notes.

**Comment by sje30** [comment]
I see what you mean, thanks!  I look forward to watching this project develop.
